api/sheet.go

```diff
@@ -0,0 +1,131 @@
+package api
+
+import (
+	"context"
+	"encoding/json"
+)
+
+// SheetVisibility is the visibility of a sheet.
+type SheetVisibility string
+
+const (
+	// Private is the sheet visibility for PRIVATE.
```

Comment:
probably should comment on the read/write behavior

Reply:
PTAL again